new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioTwoDomainsInImage.java

```diff
@@ -0,0 +1,369 @@
+    boolean deployRes = assertDoesNotThrow(
+        () -> deployHttpIstioGatewayAndVirtualservice(targetHttpFile)); 
+    assertTrue(deployRes, "Failed to deploy Http Istio Gateway/VirtualService");
+
+    Path srcDrFile = Paths.get(RESOURCE_DIR, "istio", "istio-dr-template.yaml");
+    Path targetDrFile = assertDoesNotThrow(
+        () -> generateFileFromTemplate(srcDrFile.toString(), "istio-dr1.yaml", templateMap));
+    logger.info("Generated DestinationRule file path is {0}", targetDrFile);
+
+    deployRes = assertDoesNotThrow(
+        () -> deployIstioDestinationRule(targetDrFile));
+    assertTrue(deployRes, "Failed to deploy Istio DestinationRule");
+    String clusterService2 = domainUid2 + "-cluster-" + clusterName + "." + domainNamespace2 + ".svc.cluster.local";
+    templateMap.put("NAMESPACE", domainNamespace2);
+    templateMap.put("DUID", domainUid2);
+    templateMap.put("ADMIN_SERVICE",adminServerPodName2);
+    templateMap.put("CLUSTER_SERVICE", clusterService2);
+
+    Path targetHttpFile2 = assertDoesNotThrow(
+        () -> generateFileFromTemplate(srcHttpFile.toString(), "istio-http2.yaml", templateMap));
+    logger.info("Generated Http VS/Gateway file path is {0} for domain2", targetHttpFile);
+    deployRes = assertDoesNotThrow(
+        () -> deployHttpIstioGatewayAndVirtualservice(targetHttpFile2)); 
+    assertTrue(deployRes, "Failed to deploy Http Istio Gateway/VirtualService");
+    
+    Path targetDrFile2 = assertDoesNotThrow(
+        () -> generateFileFromTemplate(srcDrFile.toString(), "istio-dr2.yaml", templateMap));
+    logger.info("Generated DestinationRule file path is {0}", targetDrFile);
+
+    deployRes = assertDoesNotThrow(
+        () -> deployIstioDestinationRule(targetDrFile2));
+    assertTrue(deployRes, "Failed to deploy Istio DestinationRule");
+
+    int istioIngressPort = getIstioHttpIngressPort();
+    logger.info("Istio Ingress Port is {0}", istioIngressPort);
+
+    String consoleUrl = "http://" + K8S_NODEPORT_HOST + ":" + istioIngressPort + "/console/login/LoginForm.jsp";
+    boolean checkConsole = 
+         checkAppUsingHostHeader(consoleUrl, domainNamespace1 + ".org");
+    assertTrue(checkConsole, "Failed to access WebLogic console on domain1");
+    logger.info("WebLogic console on domain1 is acceesible");
+    Path archivePath = Paths.get(ITTESTS_DIR, "../src/integration-tests/apps/testwebapp.war");
+    ExecResult result = null;
+    result = DeployUtil.deployUsingRest(K8S_NODEPORT_HOST, 
+        String.valueOf(istioIngressPort),
+        ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT, 
+        clusterName, archivePath, domainNamespace1 + ".org");
+    assertNotNull(result, "Application deployment failed on domain1");
+    logger.info("Application deployment on domain1 returned {0}", result.toString());
+    assertEquals("202", result.stdout(), "Deployment does not returns HTTP status code 202");
```

Comment:
Deployment did not return HTTP status code 202

Reply:
yes, deployment thru curl command returns 202 ( for a new deployment ) and 400 ( if the app is already deployed)